### PR TITLE
chore: improve all docs and best practices

### PR DESCRIPTION
**Pull request by luislucena16**
this PR focuses on improving docs and open source best practices!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,6 +234,11 @@ This plugin adds additional JSON-RPC methods to be able to interact and get data
   });
   ```
 
+## Contributions
```
Comment (excaliborr) [inline README.md:237]: We have a `# Contributors` heading towards the bottom of the readme, I would add this as a subsection there

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Code of Conduct
```
Comment (excaliborr) [inline docs/CONTRIBUTING.md:8]: If its empty just remove it plz

File: README.md (hunk 1 above)
Comment (luislucena16) [reply README.md:237]: great, I added it as a subsection also with `Code Contributors`

**Comment by luislucena16** [comment]
hey @excaliborr this PR is now mergeable! please check it

**Review by excaliborr — DISMISSED** [review]

**Comment by luislucena16** [comment]
hi @0xGorilla can you check this out? only one approve is missing!

**Review by 0xAustrian — COMMENTED** [review]
Good job. Just one comment: some parts of the text says _Defi Wonderland_, it should be just _Wonderland_

**Comment by luislucena16** [comment]
done! fixed! you can check it again @0xAustrian @excaliborr

**Comment by luislucena16** [comment]
@excaliborr @0xAustrian this PR can now be merged!

**Comment by excaliborr** [comment]
Hey @luislucena16  sorry for the late reply, we decided internally to not merge any docs/typo pr's for first time contributors, if you wanted to pick up feature work like adding support for biconomy paymaster like we talked about in the issue you are more then welcome to do so.

**Pull request closed without merging**